aspnetcore/razor-components/get-started.md

```diff
@@ -5,58 +5,167 @@ description: Learn how to get started with the Razor Components framework.
 
+By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
+
 [!INCLUDE[](~/includes/razor-components-preview-notice.md)]
 
-## Setup
+# [Visual Studio](#tab/visual-studio)
+
+Prerequisites:
+
+[!INCLUDE[](~/includes/net-core-prereqs-vs-3.0.md)]
+
+To create your first Razor Components project in Visual Studio:
+
+1. Select **File** > **New Project** > **Web** > **ASP.NET Core Web Application**.
+1. Make sure **.NET Core** and **ASP.NET Core 3.0** are selected at the top.
+1. Choose the **Razor Components** template and select **OK**.
+
+   ![New app dialog](https://msdnshared.blob.core.windows.net/media/2019/01/razor-components-template.png)
 
-Install the following:
+1. Press **Ctrl-F5** to run the app *without the debugger*. Running with the debugger (**F5**) isn't supported at this time.
+
+Congrats! You just ran your first Razor Components app!
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Prerequisites:
 
-1. [.NET Core 2.1 SDK](https://go.microsoft.com/fwlink/?linkid=873092) (2.1.500 or later).
-1. [Visual Studio 2017](https://go.microsoft.com/fwlink/?linkid=873093) (15.9 or later) with the *ASP.NET and web development* workload selected.
-1. The latest [Blazor Language Services extension](https://go.microsoft.com/fwlink/?linkid=870389) from the Visual Studio Marketplace.
-1. The Blazor templates on the command-line:
+[!INCLUDE[](~/includes/net-core-prereqs-vsc-3.0.md)]
+
+To create your first Razor Components project in Visual Studio Code:
+
+1. Execute the following command from a command shell:
 
    ```console
-   dotnet new -i Microsoft.AspNetCore.Blazor.Templates
+   dotnet new razorcomponents -o WebApplication1
    ```
 
-To create your first project from Visual Studio:
+1. Open the *WebApplication1* folder in Visual Studio Code.
 
-1. Select **File** > **New Project** > **Web** > **ASP.NET Core Web Application**.
-1. Make sure **.NET Core** and **ASP.NET Core 2.1** are selected at the top.
-1. Choose the Blazor template and select **OK**.
+1. Add a *.vscode* folder.
 
-   ![New app dialog](https://msdnshared.blob.core.windows.net/media/2018/07/new-blazor-app-dialog-0.5.0.png)
+1. Add a *tasks.json* file to the *.vscode* folder with the following content:
 
-1. Press **Ctrl-F5** to run the app *without the debugger*. Running with the debugger (**F5**) isn't supported at this time.
+   [!code-json[](get-started/samples_snapshot/3.x/tasks.json)]
+
+1. Add a *launch.json* file to the *.vscode* folder with the following content:
+
+   [!code-json[](get-started/samples_snapshot/3.x/launch.json)]
+
+1. Execute the app using the Visual Studio Code debugger.
+
+1. In a browser, navigate to `https://localhost:5001`.
+
+Congrats! You just ran your first Razor Components app!
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+.NET Core 3.0 Preview 2 isn't available on macOS at this time. .NET Core 3.0 will be supported with Visual Studio for Mac version 8.0 or later.
```

Comment:
.NET Core 3.0 Preview 2 is available on macOS. I'm not sure what the status of .NET Core 3.0 support is in VS4Mac.

Reply:
Oh, yes ... this was before my little local meltdown setting it up on the iMac.

WRT VS4Mac, not supported until VS4Mac 8.0. Not sure if that Preview is out yet. I only saw 7.7, and Matt Ward says that 3.0 isn't supported in 7.7 (which matches my experience trying it here 🙈 lol).